### DISCUSSION TITLE
elastalert2 deploy  on kubernetes

### DISCUSSION THREAD

**Discussion by baiyibing123** [jertel/elastalert2#1290] [Q&A] [answered]
If elastalert2 is deployed on kubernetes, does replicaCount have to be 1? What happens if it's 2, how do you reduce the burden of a pod

**Comment by jertel** [comment, marked as answer]
It should be 1. If you set it to 2 you will likely get duplicate alerts. There are discussions on this forum that talk about high availability with ElastAlert 2.
